src/widgets/PaperSection.js

```diff
@@ -1,69 +0,0 @@
-/* eslint-disable react/forbid-prop-types */
```

Comment:
For a long portion of this PR's development I tried to keep `PaperSection` going. But I just found frustrating for a few reasons:

- The styles between the components were inconsistent. Tweaking the base `Paper` component styles would often lead to annoying side affects in `PaperSection`.
- This was in part to do with using `defaultProps` to handle the default styles of `PaperSection`. To be fair, that sounds reasonable to do, but a common RN/React pattern (used extensively in older mobile code... so me lol) is to be able to tweak style props as needed by passing inline style objects/stylesheets. If a default prop is used to define the default look, if I make a minor tweak and pass in to that same prop I lose all styles, rather than having them combine!
- It was simply unnecessarily difficult having to edit two files with separate styling (using different approaches), for what is really 1 simple thing. It was less code/context to internalise just having one file, and made for what is still a simple and reusable component (that I believe is more maintainable).

@mark-prins also wondered if [`PaperSection` was the easiest way to go about having headers on `Paper`](https://github.com/openmsupply/mobile/pull/3337#pullrequestreview-569640311). Might have different angles?

In someways I don't like github discussions because they may seem lacking in tact if you're trying to pump out concise comments/critical feedback. Please forgive me! Happy to have discussions IRL on these sort of topics.